generators/createDisplayUnitSet/templates/shared/.sharedrc

```diff
@@ -50,14 +40,13 @@
     "css": "./css/style.css",
     "defaultFonts": [
       "OpenSans",
-      "OpenSansBold",
-      "OpenSansMedium"
+      "OpenSansBold"
```

Comment:
Maybe only use "OpenSans" (and remove OpenSansBold) as an example font as we are going to change it anyway to another project related font?

Reply:
Like I said earlier I think it's good to show how you load multiple fonts